documentation/documentation_compiler.ts

```diff
@@ -69,6 +72,18 @@ function compileExampleSnippets(example: Example): CompiledExampleSnippet[] {
   });
 }
 
+function compileExampleSnippetEmbed(exampleSnippet: ExampleSnippet) {
+  const snippetEmbedTemplate = fs.readFileSync(path.join(DocumentationRoot, 'snippet_embed_template.html'), 'utf8');
```

Comment:
Nit, you could just do this at the module level and store this as a constant, so it doesn't need to get re-parsed for each snippet.

Reply:
😅 🤦‍♂️